### DISCUSSION TITLE
TÉLÉCHARGEMENT AVEC AVAST

### DISCUSSION THREAD

**Discussion by rodlec73** [pascalaubry/papi-web#57] [General]
Bonjour,
Je n'arrive pas à télécharger "papi-web", je suis systématiquement bloqué
j'ai pourtant crée une exception (voir capture d'écran)
Avez-vous une idée?
Merci
Rodolphe, Albertville
![Capture d’écran 2024-11-28 211724](https://github.com/user-attachments/assets/bd237c26-9ca9-42ac-8f7f-13f6585a0f1e)
![Capture d’écran 2024-11-28 211826](https://github.com/user-attachments/assets/05c35cbf-aa20-4209-83c0-54a056de3892)

**Comment by Amaras** [comment]
Bonjour, l'exception concerne uniquement le fichier `papi-web-2.4.13.exe` que vous utilisez.
Je vous conseille de retirer ces exceptions.

Pour retirer le fichier de la quarantaine (et normalement faire une exception), vous devez aller dans l'onglet protection, et il devrait y avoir une option quarantaine.
À partir de là, il devrait y avoir une ligne `papi-web-2.4.13.exe`, et un bouton restaurer le fichier, peut-être caché derrière un menu à trois points.

Si vous le trouvez, merci d'envoyer des captures d'écran pour notre documentation :)

**Comment by rodlec73** [comment]
Merci pour l'info
J'ai changer d'ordinateur (qui a aussi Avast!), mais rien n'y fait!
Sur le deuxième je suis également bloqué par une détection, mais rien en quarantaine!!
Bref, il faudra que je retire Avast pour télécharger Papi-web (pour le remettre après), en espérant qu'il ne me fasse pas une détection douteuse à chaque ouverture de session!!
Merci beaucoup

**Comment by rodlec73** [comment]
Mine de rien, on avance!!
J'ai réussi à télécharger Papi-web, maintenant je n'arrive pas à l'extraire!

![image](https://github.com/user-attachments/assets/1d39d0cb-92fc-4a09-a8f6-1a0817b19ea7)
![image](https://github.com/user-attachments/assets/6ea888e1-80a0-45c3-a128-ece19dbd6099)

**Reply by Amaras** [reply]
Et en allant voir dans la quarantaine, maintenant ?
Normalement il devrait y être cette fois.

**Comment by rodlec73** [comment]
Et ces infos, vous savez ce que c'est?

![image](https://github.com/user-attachments/assets/7a73da88-6db6-4c7e-b05d-26cf383993dd)

**Reply by Amaras** [reply]
Vous essayez de mettre papi-web dans quel type de dossier ?
Normalement, il devrait être uniquement dans les dossiers utilisateurs.

**Comment by rodlec73** [comment]
Papi-web tente bien de s'extraire dans le dossier "users/..."
Les fichiers présents en quarantaine sont maintenant mis en "exception"
Du coup, j'arrive à avoir cette icone, mais impossible de l'ouvrir
![image](https://github.com/user-attachments/assets/4de6ebc5-5c82-4953-8a9a-f38e9968984b)

**Reply by Amaras** [reply]
Il faut démarrer papi-web par un des fichiers `.bat`, de préférence `server.bat`

**Comment by rodlec73** [comment]
Ca devient bon!

![image](https://github.com/user-attachments/assets/e203f342-802b-4266-9091-caee77125f78)
